products/compute/terraform.yaml

```diff
@@ -1067,6 +1078,8 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         diff_suppress_func: 'compareSelfLinkOrResourceName'
       priority: !ruby/object:Overrides::Terraform::PropertyOverride
         default_value: 1000
+      nextHopIp: !ruby/object:Overrides::Terraform::PropertyOverride
+        default_from_api: true
```

Comment:
Is next hop IP set by the API if it is not provided?

Reply:
yes incase of ilb